db/migrate/20200114125900_update_imported_from_whitehall.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class UpdateImportedFromWhitehall < ActiveRecord::Migration[6.0]
+  def change
+    TimelineEntry.where(entry_type: :imported_from_whitehall).each do |timeline_entry|
+      migration_timeline_entry = WhitehallMigration::TimelineEntry.create!(
+        entry_type: :imported_from_whitehall,
+      )
+      timeline_entry.update!(
+        entry_type: :whitehall_migration,
+        details_type: "WhitehallMigration::TimelineEntry",
+        details_id: migration_timeline_entry.id,
+      )
+    end
+  end
```

Comment:
Nice work doing this, however I don't think we're going to actually need this migration since we've never actually migrated something in production. 

I imagine given data syncs we're probably not going to need this for integration so it'd probably only benefit developers and given there'll be only a couple at most we can probably just tell them what to do.

Reply:
Very good point!  I was stuck in the world of my local environment where there are hundreds of `WhitehallMigration::DocumentImport` records.

I've removed this migration completely.